Metaverse/Source/Metaverse.Target.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
```

Comment:
is this copyright header correct for the project?

Reply:
Yes, unmodified files generated by unreal, copyright is correct at this moment.